v3/tasks/20250212-v3-mev-capture-hook/test/task.fork.ts

```diff
@@ -32,4 +33,12 @@ describeForkTest('MevCaptureHook', 'mainnet', 21832752, function () {
     const registry = await registryTask.deployedInstance('BalancerContractRegistry');
     expect(await mevCaptureHook.getBalancerContractRegistry()).to.be.eq(registry.address);
   });
+
+  it('returns default mev tax multiplier', async () => {
+    expect(await mevCaptureHook.getDefaultMevTaxMultiplier()).to.be.eq(bn('1500000000000000000000000'));
+  });
+
+  it('returns default mev tax threshold', async () => {
+    expect(await mevCaptureHook.getDefaultMevTaxThreshold()).to.be.eq(bn('300000000'));
+  });
```

Comment:
```suggestion
  });
  
  it('is enabled by default', async () => {
    expect(await mevCaptureHook.isMevTaxEnabled()).to.be.true;
  });
```
We also changed this; could add this test.